File: app/src/main/java/com/example/halilarm/Fragment2.java
package com.example.halilarm;

import android.content.Intent;
import android.graphics.Color;
import android.os.Build;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.*;
import android.widget.Button;


public class Fragment2 extends Fragment {
    Button armHo;
    @Nullable
    @Override
    public View onCreateView(@NonNull final LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        final ViewGroup view = (ViewGroup) inflater.inflate(R.layout.fragment2, container, false);
        //((MainActivity)getActivity()).updateStatusBarColor("#CC1D1D");
        armHo= view.findViewById(R.id.armHo);;
        armHo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(getActivity(),AlarmActivity.class));
            }
        });
        return view;
    }

}

File: app/src/main/java/com/example/halilarm/Fragment2.kt
package com.example.halilarm

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import androidx.fragment.app.Fragment


class Fragment2: Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment2, container, false) as ViewGroup
        var armHo = view.findViewById<Button>(R.id.armHo)
        armHo.setOnClickListener {

        }
        return view
    }
}